doc/source/conf.py

```diff
@@ -58,6 +58,10 @@
 # ray.rllib.models.lstm.py will use tf.VERSION
 sys.modules["tensorflow"].VERSION = "9.9.9"
 
+# Walkaround isinstance(o, FunctionDescriptor) raises
+# TypeError: isinstance() arg 2 must be a type or tuple of types
```

Comment:
Would you mind explain a bit on what this means?

Reply:
sphinx needs to mock ray._raylet, but the auto generated mock FunctionDescriptor can not pass isinstance check. So, here make mock FunctionDescriptor is a type for sphinx.